test/tbb/test_task_group.cpp

```diff
@@ -1222,3 +1222,124 @@ TEST_CASE("task_handle cannot be scheduled into other task_group of the same con
 }
 
 #endif // TBB_USE_EXCEPTIONS
+
+#if __TBB_PREVIEW_TASK_GROUP_EXTENSIONS
+TEST_CASE("test task_tracker") {
```

Comment:
Case that I think are missing:
-  is concurrent creation / destruction of trackers to same handle
- creation of tracker to deferred task but handle is destroyed without being run